cibuild.cmd

```diff
@@ -1,5 +1,148 @@
-@echo off
+@if not defined _echo echo off
+setlocal
 
-echo cibuild.cmd invoked from the xplat branch--calling RebuildWithLocalMSBuild.cmd
+:parseArguments
+if "%1"=="" goto doneParsingArguments
+if /i "%1"=="--scope" set SCOPE=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--target" set TARGET=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--host" set HOST=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--skip-bootstrap" set BOOTSTRAP_ONLY=true&& shift && goto parseArguments
```

Comment:
Agreed. I was thinking about doing that in a future commit. Right now the external interface matches between this and cibuild.sh.